src/nvim/ex_docmd.h

```diff
@@ -19,6 +19,20 @@
 #define EXMODE_NORMAL           1
 #define EXMODE_VIM              2
 
+/// The scope of a command.
+///
+/// The lower a number, the deeper the scope.
+typedef enum {
+  kCdScopeWindow,  ///< Affects one window.
+  kCdScopeTab,     ///< Affects on tab page.
+  kCdScopeEditor,  ///< Affects the entire instance of NeoVim.
```

Comment:
Oh, you mean the user-manual documentation. Makes, sense, I'll rename it to global then.